source/projects/muleify.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Muleify
+repo: AlexanderElias/muleify
+homepage: https://github.com/AlexanderElias/muleify
+language: JavaScript
+license: MPL-2.0
+templates: Comments
+description: "A zero configuration static site generator and asset preprocessor compiler."
+---
+
+# Muleify #
```

Comment:
Can you remove this title. It gets duplicated, see: https://deploy-preview-226--staticgen.netlify.com/muleify

<img width="944" alt="screenshot 2017-03-01 21 53 56" src="https://cloud.githubusercontent.com/assets/5713670/23494777/9e26cb82-fec9-11e6-8689-7ebef70cdd16.png">